Package.swift

```diff
@@ -24,6 +24,10 @@ let package = Package(
         .target(
             name: "Library"
         ),
+        .testTarget(
+            name: "LibraryXCTests",
```

Comment:
```suggestion
            name: "LibraryXCTests",
```

maybe just Tests? We'll adopt swift-testing here shortly I assume

Reply:
In fact, I chose this name because the PR includes both XCTest and swift-testing!